src/main.ts

```diff
@@ -12,10 +14,14 @@ Vue.config.productionTip = false;
 Vue.use(AsyncComputed);
 Vue.use(VueCompositionApi);
 
-new Vue({
-  render: (h) => h(App),
-  router,
-  // Necessary to correctly use direct-vuex
-  store: store.original,
-  vuetify,
-}).$mount('#app');
+oauthClient.maybeRestoreLogin().then(() => {
+  Object.assign(api.client.axios.defaults.headers.common, oauthClient.authHeaders);
```

Comment:
@zachmullen, how did you know to access the axios instance's persistent request headers in this way? Is this "safe"?

Reply:
I believe I looked it up and this was the recommended approach. It seems to work -- I'm not sure under exactly what conditions it's safe or not.